server/service/src/user_account.rs

```diff
@@ -148,9 +148,13 @@ impl<'a> UserAccountService<'a> {
             )
     }
 
-    pub fn find_user(&self, user_id: &str) -> Result<Option<User>, RepositoryError> {
+    pub fn find_user(&self, user_id: &str, site_id: i32) -> Result<Option<User>, RepositoryError> {
         let repo = UserRepository::new(self.connection);
-        repo.query_one(UserFilter::new().id(EqualFilter::equal_to(user_id)))
```

Comment:
I think this method is only used in the context of a certain site and you can't lookup users from different sites. For this reason I would move the site_id lookup from graphql into this method.

Reply:
i agree, generally speaking we don't want to use repositories in graphql layer